### ISSUE TITLE
Inconsistent file folder structure in documentation

### ISSUE THREAD

**Issue by szeliski** [naver/kapture#30]
If you look at the description of the file folder hierarchy at the top of the `kapture: data format v1.1` document at https://github.com/naver/kapture/blob/main/kapture_format.adoc, it shows the images as all living in sub-folders of the sensors/records_data with both map (sic) and query folders inside that one records_data folder.

But if you look at the folder structure in the localization datasets downloaded with `kapture_download_dataset`, the `mapping` and `query` are top-level folders within which the `sensors` folders live.

Is the on-line documentation out of date with the new file folder structure?

Since the map(ping) and query images and meta-data are in separate top-level folders (and hence, presumably would be the created `reconstruction` folders), what is the recommended strategy for matching mapping vs. query images and storing these matches?

Thanks.

**Comment by jujumo** [comment]
Hi Richard,

Thank you for bringing this to my attention.
You are right, this discrepancy is misleading.

The kapture format does not allow hadoc split of mapping and query.
That is why we provide mapping and query are top-level folders within the downloadable datasets.
In case of the documentation, the purpose was just to demonstrate that you can have subfolders under `records_data/`, not only images. But there is no split in possible in calibration or reconstruction.

I agree, this is misleading, and I'll choose another example than map/query to demonstrate the `records_data/` structure.

**Comment by humenbergerm** [comment]
Let me add something about the recommended strategy:
We usually use [this folder structure](https://github.com/naver/kapture-localization/blob/main/doc/tutorial.adoc#recommended-dataset-structure) for our experiments.
For example, there can be one kapture for mapping and one for query. The images could be stored in one location (here you could have both, mapping and query images) and the records_data folders could be symbolic links to this location. Which images are actually used in the respective kaptures is defined in their records_camera.txt files.

Same for local and global features as well as for matches. They can all be stored in one shared location and the kaptures itself get symbolic links to these locations.  Storing reconstruction data (features, matches) in one common location (and linking them when needed) has the advantage that this data will be reused. For example if you do feature matching, only missing matches will be computed and existing ones will be reused.

In the [pipeline examples](https://github.com/naver/kapture-localization/blob/main/doc/tutorial.adoc#examples), the recommended data structure will be used and this hopefully helps understanding our data handling strategy.
In summary, the scripts use one location to store all datasets downloaded and they create one folder per dataset to store the reconstruction data (features and matches) and experiment results.

**Comment by szeliski** [comment]
Thanks.  This clarifies everything nicely, especially the [folder structure](https://github.com/naver/kapture-localization/blob/main/doc/tutorial.adoc#recommended-dataset-structure) link.  I think that just putting a prominent pointer to this documentation where you have the original (somewhat outdated) folder structure should be enough to clear things up.
